google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.java

```diff
@@ -119,24 +119,6 @@ public long getInflightWaitSeconds() {
     return this.schemaAwareStreamWriter.getInflightWaitSeconds();
   }
 
-  /**
-   * Sets the missing value interpretation map for the JsonStreamWriter. The input
-   * missingValueInterpretationMap is used for all append requests unless otherwise changed.
-   *
-   * @param missingValueInterpretationMap the missing value interpretation map used by the
-   *     JsonStreamWriter.
-   */
-  public void setMissingValueInterpretationMap(
-      Map<String, AppendRowsRequest.MissingValueInterpretation> missingValueInterpretationMap) {
-    this.schemaAwareStreamWriter.setMissingValueInterpretationMap(missingValueInterpretationMap);
-  }
-
-  /** @return the missing value interpretation map used for the writer. */
-  public Map<String, AppendRowsRequest.MissingValueInterpretation>
-      getMissingValueInterpretationMap() {
```

Comment:
Should we keep the get method?

Reply:
It will be less disruptive for the customers. Done.